railties/lib/rails/generators/rails/plugin/plugin_generator.rb

```diff
@@ -92,13 +93,10 @@ def test
       end
     end
 
-    PASSTHROUGH_OPTIONS = [
-      :skip_active_record, :skip_active_storage, :skip_action_mailer, :skip_javascript, :skip_action_cable, :skip_sprockets, :database,
-      :api, :quiet, :pretend, :skip
-    ]
+    DUMMY_IGNORE_OPTIONS = %i[dev edge master template]
 
     def generate_test_dummy(force = false)
-      opts = (options.dup || {}).keep_if { |k, _| PASSTHROUGH_OPTIONS.map(&:to_s).include?(k) }
+      opts = options.transform_keys(&:to_sym).except(*DUMMY_IGNORE_OPTIONS)
```

Comment:
I'm not sure what the consequences of this are, but using `transform_keys` or `except` here will return a plain hash instead of a `Thor::CoreExt::HashWithIndifferentAccess`, as `slice` did before https://github.com/rails/rails/commit/20df3f778649d50498a5c04b98abd2379f56b31c.

Reply:
Right, I think there was some confusion over what `invoke` expected.  Calling `slice` on `Thor::CoreExt::HashWithIndifferentAccess` will return a normal Hash with String keys, but `invoke` [expects a Hash with Symbol keys](https://github.com/erikhuda/thor/blob/v1.0.1/lib/thor/invocation.rb#L100-L102).  A `Thor::CoreExt::HashWithIndifferentAccess` can be used because it accepts Symbol keys, but it is not specifically required.

Reply:
Got it. Thanks for the explanation!